homeassistant/components/imap_email_content/sensor.py

```diff
@@ -53,7 +54,7 @@
 )
 
 
-def setup_platform(
+async def async_setup_platform(
```

Comment:
Is the imap client here really async safe?

We can add a job with the issue processing from sync context with `hass.add_job`.